cdc/generation.cc

```diff
@@ -292,6 +295,22 @@ future<utils::chunked_vector<mutation>> get_cdc_generation_mutations(
         co_await coroutine::maybe_yield();
     }
 
+    total_size_estimate += size_estimate;
+
+    // Copy mutations n times, where n is picked that the memory size of all mutations exceeds `max_command_size`.
```

Comment:
"is picked that" -> "is picked so that"
"the memory size of all mutations exceeds" -> "the memory size of all mutations together exceeds" (to make it clear that it's not each mutation size exceeding `max_command_size`; it's their total)

Reply:
Done.